lib/RAST_SDK/AnnotationUtils.pm

```diff
@@ -1,6 +1,4 @@
-# -*- perl -*-
-
-package anno_utils;
+package RAST_SDK::AnnotationUtils;
```

Comment:
renamed to conform to perl class naming standards, and moved into the RAST_SDK directory